sys-apps/sum-driver/files/makefile

```diff
@@ -0,0 +1,21 @@
+obj-m := sum_bios.o
+
+KVERSION := $(shell uname -r)
+KDIR := /lib/modules/$(KVERSION)/build
+PWD := $(shell pwd)
+
+EXTRA_CFLAGS = $(CFLAGS) -Wno-incompatible-pointer-types
+EXTRA_LDFLAGS = $(LDFLAGS)
+
+clean:
+	$(MAKE) -C $(KDIR) M=$(PWD) clean
+
+default:
+	$(MAKE) -C $(KDIR) M=$(PWD) modules
+
+install:
+	$(MAKE) -C $(KDIR) M=$(PWD) modules_install
+
+load:
```

Comment:
What for? If you need to bundle a Makefile for it, at least make it minimal.

Reply:
I am sorry, but I never have written a makefile for modules. I just took a default from portage, where I assumed, this all should be in. Updated now for dropping clean and load.